src/js/_enqueues/wp/updates.js

```diff
@@ -2462,4 +2471,151 @@
 		 */
 		$( window ).on( 'beforeunload', wp.updates.beforeunload );
 	} );
+
+	/**
+	 * Click handler for enabling and disabling plugin and theme auto-updates.
+	 *
+	 * @since 5.5.0
+	 *
+	 * TODO: this needs to be refactored to be like the remainder of the handlers
+	 *       in this file.
+	 */
+	$( document ).ready( function() {
+		$( $document ).on( 'click', '.column-auto-updates a.toggle-auto-update, .theme-overlay a.toggle-auto-update', function( event ) {
+			var data, asset, type, $parent;
+			var $anchor = $( this ),
+				action = $anchor.attr( 'data-wp-action' ),
```

Comment:
All the `$.attr( 'data-xyz' )` should be changed to use `$.data( 'xyz' )`

```suggestion
				action = $anchor.data( 'wp-action' ),
```